ssl/statem/statem_clnt.c

```diff
@@ -495,6 +495,10 @@ static WRITE_TRAN ossl_statem_client13_write_transition(SSL_CONNECTION *s)
         /* Fall through */
 
     case TLS_ST_CW_END_OF_EARLY_DATA:
+        if (SSL_CONNECTION_IS_DTLS13(s)) {
+            /* If we are done with early data we need to clean up Epoch 1 messages sent*/
```

Comment:
If we don't call dtls1_clear_sent_buffer when leaving epoch 1 we will then leak the write epoch 1 record layer. I don't know if this is the right spot for this. Please let me know.